### PR TITLE
Registrar: Updated whois readaction wording

### PR DESCRIPTION
**Pull request by marciocloudflare**

### CODE REVIEW

File: products/registrar/src/content/domain-registration/whois-redaction.md [hunk 1]
```diff
@@ -17,11 +17,19 @@ However, broadcasting the registrant contact information, via the WHOIS service,
 --------
 
 ## What is WHOIS redaction?
-WHOIS redaction removes all contact information categorized as personal data from the published WHOIS record for a domain (registrant name, email address, postal address). Fields will read “Data Redacted”. The nameserver, domain lock information, and date records for a domain are still available publicly.
+
+WHOIS redaction removes most contact information categorized as personal data from the published WHOIS record for a domain (registrant name, email address, postal address). Fields will read “Data Redacted”. The nameserver, domain lock information, and date records for a domain are still available publicly.  The following fields will continue to show in WHOIS:
```
Comment (abracchi-tw) [inline products/registrar/src/content/domain-registration/whois-redaction.md]: I would move the parenthesis right after the element it elaborates on (in this case, personal data) what do you think about

`WHOIS redaction removes most contact information categorized as personal data (registrant name, email address, postal address) from the published WHOIS record for a domain .`

File: products/registrar/src/content/domain-registration/whois-redaction.md [hunk 2]
```diff
@@ -17,11 +17,19 @@ However, broadcasting the registrant contact information, via the WHOIS service,
...
+
+* Registrant State/Province 
+* Registrant Country.
 
 Cloudflare still maintains the authoritative, unredacted, record of your WHOIS data. You can modify this information at any time in the domain overview tab of the Cloudflare dashboard.
 
-Note: WHOIS redaction is not the same as WHOIS Privacy. WHOIS Privacy replaces your information with proxy contact information. Redaction removes it altogether.
+<Aside type="note" header="Note">
+
+WHOIS redaction is not the same as WHOIS Privacy. WHOIS Privacy replaces your information with proxy contact information. Redaction removes it altogether.
```
Comment (abracchi-tw) [inline products/registrar/src/content/domain-registration/whois-redaction.md]: nit:
```suggestion
WHOIS redaction is not the same as WHOIS Privacy. WHOIS Privacy replaces your information with proxy contact information, while redaction removes it altogether.
```

Comment (abracchi-tw) [reply products/registrar/src/content/domain-registration/whois-redaction.md]: any reason why `redaction` is lowercase and `privacy` is capitalized?

**Review by abracchi-tw — APPROVED** [review]

File: products/registrar/src/content/domain-registration/whois-redaction.md (hunk 2 above)
Comment (marciocloudflare) [reply products/registrar/src/content/domain-registration/whois-redaction.md]: none

File: products/registrar/src/content/domain-registration/whois-redaction.md (hunk 1 above)
Comment (marciocloudflare) [reply products/registrar/src/content/domain-registration/whois-redaction.md]: Sounds good!

**Pull request merged**